File: src/targets/node/unirest/fixtures/full.js
const unirest = require('unirest');

const req = unirest('POST', 'http://mockbin.com/har');

const CookieJar = unirest.jar();
CookieJar.add('foo=bar', 'http://mockbin.com/har');
CookieJar.add('bar=baz', 'http://mockbin.com/har');
req.jar(CookieJar);

req.query({
  foo: [
    'bar',
    'baz'
  ],
  baz: 'abc',
  key: 'value'
});

req.headers({
  accept: 'application/json',
  'content-type': 'application/x-www-form-urlencoded'
});

req.form({
  foo: 'bar'
});

req.end(function (res) {
  if (res.error) throw new Error(res.error);

  console.log(res.body);
});

File: src/targets/java/unirest/fixtures/full.java
HttpResponse<String> response = Unirest.post("https://httpbin.org/anything?foo=bar&foo=baz&baz=abc&key=value")
  .header("cookie", "foo=bar; bar=baz")
  .header("accept", "application/json")
  .header("content-type", "application/x-www-form-urlencoded")
  .body("foo=bar")
  .asString();